src/components/layout/ContactsSelectionBar.jsx

```diff
@@ -43,26 +44,37 @@ class ContactsSelectionBar extends Component {
                 description = 'delete-confirmation.description-mixed'
 
               showModal(
-                <Modal
-                  into="body"
+                <ConfirmDialog
+                  open={true}
+                  onClose={hideModal}
                   title={t('delete-confirmation.title', {
                     smart_count: selection.length
                   })}
-                  description={t(description, {
+                  content={t(description, {
                     smart_count: selection.length
                   })}
-                  primaryText={t('delete')}
-                  primaryType="danger"
-                  primaryAction={async () => {
-                    await Promise.all(
-                      selection.map(contact => deleteContact(client, contact))
-                    )
-                    clearSelection()
-                    hideModal()
-                  }}
-                  secondaryText={t('cancel')}
-                  secondaryAction={() => hideModal()}
-                  dismissAction={hideModal}
+                  actions={
+                    <>
+                      <Button
+                        theme="secondary"
+                        label={t('cancel')}
+                        onClick={hideModal}
+                      />
+                      <Button
+                        theme="danger"
+                        label={t('delete')}
+                        onClick={async () => {
+                          await Promise.all(
+                            selection.map(contact =>
+                              deleteContact(client, contact)
+                            )
+                          )
+                          clearSelection()
+                          hideModal()
```

Comment:
While the promise is running, it'd be interesting to have the busy prop set on the button, otherwise nothing happens while the promise is in flight.

Reply:
Oui bonne idée, on peut le faire sur une prochaine PR, je le mets en issue après avoir un peu regardé sans succès